corehq/apps/users/static/users/js/roles.js

```diff
@@ -72,6 +72,200 @@ hqDefine('users/js/roles',[
+                        showOption: root.webAppsPrivilege,
+                        editPermission: self.permissions.access_web_apps,
+                        viewPermission: null,
+                        text: gettext("<strong>Web Apps</strong> &mdash; use Web Apps for online data entry"),
+                        showEditCheckbox: true,
+                        editCheckboxLabel: "edit-web-apps-checkbox",
+                        showViewCheckbox: false,
+                        viewCheckboxLabel: "view-web-apps-checkbox",
+                        screenReaderEditAndViewText: gettext("Access Web Apps"),
+                        screenReaderViewOnlyText: null,
+                        showAllowCheckbox: false,
+                        allowCheckboxText: null,
+                        allowCheckboxId: null,
+                        allowCheckboxPermission: null,
+                    },
+                    {
+                        showOption: true,
+                        editPermission: self.permissions.access_api,
+                        viewPermission: null,
+                        text: gettext("<strong>Access APIs</strong> &mdash; use CommCare HQ APIs to read and update data. Specific APIs may require additional permissions."),
+                        showEditCheckbox: true,
+                        editCheckboxLabel: "edit-apis-checkbox",
+                        showViewCheckbox: false,
+                        viewCheckboxLabel: "view-apis-checkbox",
+                        screenReaderEditAndViewText: gettext("Access APIs"),
+                        screenReaderViewOnlyText: null,
+                        showAllowCheckbox: false,
+                        allowCheckboxText: null,
+                        allowCheckboxId: null,
+                        allowCheckboxPermission: null,
+                    },
+                    {
+                        showOption: self.permissions.access_all_locations,
+                        editPermission: self.permissions.edit_apps,
+                        viewPermission: null,
+                        text: gettext("<strong>Applications</strong> &mdash; modify or view the structure and configuration of all applications."),
+                        showEditCheckbox: true,
+                        editCheckboxLabel: "edit-apps-checkbox",
+                        showViewCheckbox: false,
+                        viewCheckboxLabel: "view-apps-checkbox",
+                        screenReaderEditAndViewText: gettext("Edit & View Apps"),
+                        screenReaderViewOnlyText: null,
+                        showAllowCheckbox: false,
+                        allowCheckboxText: null,
+                        allowCheckboxId: null,
+                        allowCheckboxPermission: null,
+                    },
+                    {
+                        showOption: self.permissions.access_all_locations,
+                        editPermission: function () {return false},
```

Comment:
Does this work in the places that refer to `editPermission` rather than `editPermission()`